integration_test/tests/wallet.rs

```diff
@@ -622,6 +622,17 @@ fn wallet__wallet_lock() {
     let _: () = node.client.wallet_lock().expect("walletlock");
 }
 
+#[test]
+fn wallet__wallet_passphrase_and_lock() {
```

Comment:
Is there a reason this is called `wallet_passphrase_and_lock` and not just `wallet_passphrase`?